Transition animation

s2jack opened the pull request:
Add animations and transitions to the elements with pair programming.

codeWithHyder committed:
first commit to desktp version one website

codeWithHyder committed:
fixed all stylelint errors

codeWithHyder committed:
style lint errors fixed

codeWithHyder committed:
style lont errors fixed

codeWithHyder committed:
style lint errors fixed

codeWithHyder committed:
stylelint errors

codeWithHyder committed:
style lint errors

codeWithHyder committed:
stylelint errors

codeWithHyder committed:
stylelint errors

codeWithHyder committed:
style lint errors fixed

codeWithHyder committed:
css error fixed

codeWithHyder committed:
css errors

codeWithHyder committed:
stylelint errors

codeWithHyder committed:
css error

codeWithHyder committed:
added what was asked by reviewer

codeWithHyder committed:
fixed stylelint errors

codeWithHyder committed:
stylelint errors

codeWithHyder committed:
stylelint errors

codeWithHyder committed:
css errors

codeWithHyder committed:
css errors

codeWithHyder committed:
css stylelint errors

codeWithHyder committed:
stylelint errors fixed

codeWithHyder committed:
errors fixed

codeWithHyder committed:
stylelint errors

codeWithHyder committed:
added hover effect on cards

codeWithHyder committed:
stylelint errors

codeWithHyder committed:
add transition to desktop version day 2

s2jack committed:
Add transition hover effect to menu items

Add transition hover effect to menu items

s2jack committed:
Fix linter errors

Fix linter errors

s2jack committed:
Fix linter errors

Fix linter errors

codeWithHyder committed:
add scale and transition to nave elements

codeWithHyder committed:
add transition and scale to nav elements

codeWithHyder committed:
add scale and transition to menu elements

codeWithHyder committed:
stylelint errors fixed

codeWithHyder committed:
style lint errors

codeWithHyder committed:
stylelint errors

codeWithHyder committed:
complete transition to desktop version

codeWithHyder committed:
add transition and transform to cards

codeWithHyder committed:
fix stylelint errors

codeWithHyder committed:
stylelint errors fixed

codeWithHyder committed:
stylelint errors fixed

codeWithHyder committed:
stylelint errors

codeWithHyder committed:
style lint errors

codeWithHyder committed:
style lint errors

codeWithHyder committed:
sle

codeWithHyder committed:
stylelint errors

codeWithHyder committed:
stylelint errors fixed

codeWithHyder committed:
style lint errors fixed

codeWithHyder committed:
style lint errors

codeWithHyder committed:
conflict resolved

codeWithHyder committed:
eol white space errors

codeWithHyder committed:
conflict with master fixed

codeWithHyder committed:
style lint errors

codeWithHyder committed:
css eol space errors

codeWithHyder committed:
Update README.md

codeWithHyder committed:
css errors

codeWithHyder committed:
css erors

codeWithHyder committed:
css sylelint error

codeWithHyder committed:
css errors

codeWithHyder committed:
css errors

codeWithHyder committed:
css errors

codeWithHyder committed:
css errors

codeWithHyder committed:
css errors

codeWithHyder committed:
css errors

codeWithHyder committed:
css style lint error

codeWithHyder committed:
css errors

codeWithHyder committed:
css errors

codeWithHyder committed:
css errors

codeWithHyder committed:
menu item fixed

andgarzonmal reviewed (approved):
Hi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<img src="https://media2.giphy.com/media/YKdm91XONLTaw/giphy-downsized-medium.gif"/>

## Highlights
- No linter errors ✔️ 
- Good implementations  ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

codeWithHyder committed:
readme file updated

codeWithHyder committed:
stlylint errors

codeWithHyder committed:
css style lint errors fixed

codeWithHyder committed:
css errors

codeWithHyder committed:
css stylelint errors

codeWithHyder committed:
css errors

codeWithHyder committed:
css errors

codeWithHyder committed:
css stylelint errors

s2jack committed:
Fix linter errors

Try to fix the linter error. 

There is no whitespace but there is a styleint error.

s2jack committed:
Fix linter errors

Fix linter errors

codeWithHyder committed:
css stylelint hite space ignored

codeWithHyder committed:
Merge branch 'transition-animation' of https://github.com/codeWithHyder/portfolionew into transition-animation

codeWithHyder committed:
css errors
